src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -788,4 +788,14 @@ public String getQueryContext() {
   public void setQueryContext(String queryContext) {
     this.queryContext = queryContext;
   }
+
+  /**
+   * If true, JDBC will enable returning TIMESTAMP_WITH_TIMEZONE as column type, otherwise it will
+   * not. This function will always return true for JDBC client, so that the client JDBC will not
+   * have any behavior change. Stored proc JDBC will override this function to return the value of
+   * SP_JDBC_ENABLE_TIMESTAMP_WITH_TIMEZONE from server for backward compatibility.
+   */
+  public boolean getEnableTimestampReturnType() {
```

Comment:
What about renaming the function like  `getEnableReturnTimestampWithTimeZone`

Reply:
ack